iris_grib/_grib_cf_map.py

```diff
@@ -16,11 +16,11 @@
 # along with iris-grib.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Could you also edit the license header to be the same as the other files:
https://github.com/SciTools/iris-grib/blob/master/iris_grib/__init__.py#L1-L5

We have dropped the old complex license header.

This change will also apply to iris